views/layouts/main.blade.php

```diff
@@ -82,6 +82,10 @@
             window['{{ config('twill.js_namespace') }}'].twillLocalization = {!! json_encode($twillLocalization) !!};
             window['{{ config('twill.js_namespace') }}'].STORE = {};
             window['{{ config('twill.js_namespace') }}'].STORE.form = {};
+            window['{{ config('twill.js_namespace') }}'].STORE.config = {
+                publishDate24Hr: {{config('twill.publish_date_24h') ? 'true' : 'false'}},
```

Comment:
It doesn't look like this isn't being used in Vue components, so we should be able to remove it.